src/runtime/analytics-service.js

```diff
@@ -89,6 +91,12 @@ export class AnalyticsService {
     this.eventManager_.registerEventListener(
       this.handleClientEvent_.bind(this)
     );
+
+    /** @private {!number} */
+    this.sent_ = 0;
```

Comment:
Could you describe this more with a comment and/or variable rename?

Reply:
Done